docs/deployment/react.md

```diff
@@ -26,6 +26,25 @@ npm i react react-dom mobx styled-components core-js
 import { RedocStandalone } from 'redoc';
 ```
 
+### IE polyfills
```

Comment:
question: I'm not sure if this fully works? I thought we stopped IE11 support earlier this year?  (cc: @RomanHotsiy )

Reply:
I have removed the IE polyfills section per @swapnilogale's guidance